src/Components/LandingPage/LandingPage.js

```diff
@@ -32,122 +37,149 @@ export const LandingPage = () => {
-              from the console.
-            </p>
-            <p>
-              Link custom repositories and build any supported image with custom
-              content.
-            </p>
-          </Alert>
-        )}
-        {isBeta() && (
-          <ExpandableSection
-            className="pf-m-light pf-u-mb-xl expand-section"
-            toggleText="Help get started with Preview features"
-            onToggle={setShowHint}
-            isExpanded={showHint}
-            displaySize="large"
+            <Button
+              variant="plain"
+              aria-label="About image builder"
+              className="pf-u-pl-sm header-button"
+            >
+              <HelpIcon />
+            </Button>
+          </Popover>
+          <OpenSourceBadge repositoriesURL="https://www.osbuild.org/guides/image-builder-service/architecture.html" />
+        </PageHeader>
+        {edgeParityFlag ? (
+          <Tabs
+            className="pf-c-tabs pf-c-page-header pf-c-table"
+            activeKey={activeTabKey}
+            onSelect={handleTabClick}
           >
-            <p className="pf-u-pb-sm">
-              For help getting started, access the quick starts for our Preview
-              features.
-            </p>
-            <p className="pf-u-pt-sm">
-              <Button
-                icon={<ArrowRightIcon />}
-                iconPosition="right"
-                variant="link"
-                isInline
-                component="a"
-                onClick={activateQuickstart('insights-launch-aws')}
-                className="pf-u-font-weight-bold"
-              >
-                Launch an AWS Image
-              </Button>
-            </p>
-            <p className="pf-u-pt-sm">
-              <Button
-                icon={<ArrowRightIcon />}
-                iconPosition="right"
-                variant="link"
-                isInline
-                component="a"
-                onClick={activateQuickstart('insights-launch-azure')}
-                className="pf-u-font-weight-bold"
-              >
-                Launch an Azure Image
-              </Button>
-            </p>
-            <p className="pf-u-pt-sm">
-              <Button
-                icon={<ArrowRightIcon />}
-                iconPosition="right"
-                variant="link"
-                isInline
-                component="a"
-                onClick={activateQuickstart('insights-custom-repos')}
-                className="pf-u-font-weight-bold"
-              >
-                Build an Image with Custom Content
-              </Button>
-            </p>
-          </ExpandableSection>
+            <Tab
+              eventKey={0}
+              title={<TabTitleText>Traditional (RPM - DNF)</TabTitleText>}
+            >
+              {traditionalImageList}
+            </Tab>
+            <Tab
+              eventKey={1}
+              title={<TabTitleText>Immutable - (OSTree)</TabTitleText>}
```

Comment:
I think that this should be `Immutable (OSTree)` to match the designs. Basically, remove the `-`. :sweat_smile:

Reply:
thank you @lucasgarfield removed